Gusto/gusto.github.io
# Fix payrolls totals documentation to have a correct check_amount

## Description

The previous documented totals did not have a valid check_amount. Brian Vu requested we update the docs to reflect reality more closely.

## Changes

- Fix payrolls totals documentation to have a correct check_amount

## Testing

* Execute `bundle exec jekyll serve`
* Visit http://localhost:4000/v1/payrolls
* Observe changed totals, especially that check_amount is nonzero
